Formula/ots.rb

```diff
@@ -0,0 +1,17 @@
+class Ots < Formula
+  desc "🔐 Share end-to-end encrypted secrets with others via a one-time URL"
+  homepage "https://github.com/sniptt-official/ots-cli"
+  url "https://github.com/sniptt-official/ots-cli/releases/download/v0.0.5/ots-cli_0.0.5_darwin_amd64.tar.gz"
+  version "0.0.5"
+  sha256 "36131e5254e5f8d45fd88c379d8791fd37a94dc6fef5ae1b324849c5ad338a2f"
+  license "Apache-2.0"
+
+  def install
+    bin.install "ots-cli" => "ots"
+  end
+
+  test do
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.

Reply:
Sorry, my bad, I found this example in another formula somewhere (might not have been in core) so thought it would be sufficient - I will fix this.